src/main.js

```diff
@@ -1,5 +1,8 @@
 import Vue from 'vue'
 import App from './App.vue'
+import './assets/styles/index.css'
```

Comment:
@21adrian1996 I think this should be importable through the `~` alias
https://cli.vuejs.org/guide/html-and-static-assets.html#url-transform-rules

Reply:
@thetractor 
Did not manage to get this work...